lte/protos/subscriberdb.proto

```diff
@@ -204,34 +204,70 @@ service SubscriberDB {
 // SubscriberDB cloud service definition.
 // --------------------------------------------------------------------------
 service SubscriberDBCloud {
+  // CheckSubscribersInSync checks whether the client subscribers flat digest
+  // is up-to-date.
+  rpc CheckSubscribersInSync (CheckSubscribersInSyncRequest) returns (CheckSubscribersInSyncResponse) {}
+  // SyncSubscribers returns the changeset of subscribers between client and
+  // cloud if the set is small; if the set is large, returns signal to resync.
+  rpc SyncSubscribers (SyncSubscribersRequest) returns (SyncSubscribersResponse) {}
   // ListSubscribers lists pages of subscribers stored.
   rpc ListSubscribers (ListSubscribersRequest) returns (ListSubscribersResponse) {}
 }
 
-message SubscribersDigest {
+message Digest {
   // md5_base64_digest is a base64-encoded MD5 digest of a set of subscriber protos.
   string md5_base64_digest = 1;
 }
```

Comment:
Move this to the end: general order is
1. rpc defns
2. req/res messages
3. all other messages